M2/Macaulay2/tests/engine/gb-bench.m2

```diff
@@ -1,42 +1,40 @@
 -- -*- coding: utf-8 -*-
+needs "raw-util.m2"
```

Comment:
The changes to this file seem unrelated to the topic of the pull request.  Were they intended?

Reply:
Yes, I fixed some internal gb tests that I figured out how to correct when I was working on `m2/gb.m2`